cmd/siac/main.go

```diff
@@ -118,5 +174,88 @@ func main() {
 		} else {
 			fmt.Printf("%s SC\n", balance.Siacoins)
 		}
+
+	case walletAddressCmd:
+		if len(args) != 0 {
+			cmd.Usage()
+			return
+		}
+		address, err := getClient().WalletAddress()
+		check("couldn't get address:", err)
+		fmt.Println(address.Address)
+
+	case walletAddressesCmd:
+		if len(args) != 0 {
+			cmd.Usage()
+			return
+		}
+		addresses, err := getClient().WalletAddresses()
+		check("couldn't get address:", err)
+		for _, address := range addresses.Addresses {
+			fmt.Println(address)
+		}
+	case walletTransactionsCmd:
+		if len(args) != 0 {
+			cmd.Usage()
+			return
+		}
+		transactions, err := getClient().Transactions()
+		check("couldn't get transactions:", err)
+
+		var parsedTxnID types.TransactionID
+		if flagg.IsDefined(cmd, "id") {
+			check("couldn't parse transaction ID:", json.Unmarshal([]byte(`"`+*transactionID+`"`), &parsedTxnID))
+		}
+
+		w := tabwriter.NewWriter(os.Stdout, 8, 8, 0, '\t', 0)
```

Comment:
`tabwriter.NewWriter` is one of the worst APIs in the stdlib lol. I can never remember what the args do. It looks like we always used `tabwriter.NewWriter(os.Stdout, 0, 0, 2, ' ', 0)` in `siac`, though.